### ISSUE TITLE
Bug: first installation and restart in AS is not working

### ISSUE THREAD

**Issue by Goooler** [ComposeGears/Valkyrie#32]
Search and install `Valkyrie` via plugin market in and restart Android Studio, you can see the plugin is not installed. Must have to install this plugin and open a project first, you'll see the sidebar icon, and then the plugin installed state will be retained after later restarts.

```
Android Studio Ladybug | 2024.1.3 Canary 1
Build #AI-241.18034.62.2413.12081498, built on July 12, 2024
Runtime version: 21.0.3+-76852501-b509.4 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.5
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 10240M
Cores: 11
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  terminal.new.ui=true
Non-Bundled Plugins:
  io.github.composegears.valkyrie (0.3.0)
  com.jetbrains.kmm (0.8.2(241)-6)
  org.jetbrains.compose.desktop.ide (1.6.2)
  com.github.copilot (1.5.16.6140)
```

**Comment by egorikftp** [comment]
Reproduced on all IDEA versions except 2024.2 BETA (EAP channel)

<img width="417" alt="image" src="https://github.com/user-attachments/assets/cb24cd22-ac82-4763-9f46-cbe7f2b1644b">


I guess the issue reproduces due to Valkyrie is a [Dynamic plugin](https://plugins.jetbrains.com/docs/intellij/dynamic-plugins.html) and JetBrains team make some improvement in latest eap version.

@Goooler If you have time, could you please check the behavior on the latest EAP version?

**Comment by Goooler** [comment]
I tried. Yeah, this has been fixed in 2024.2 Beta.

**Comment by egorikftp** [comment]
Will close for now, since it's 3rd party issue
